src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -95,21 +104,18 @@ export const AssetTransfer: React.FC<IAssetTransferProps> = (props) => {
                 />
                 <AssetTransferAddress txRole="recipient" participant={recipient} addressUrl={recipientUrl} />
             </div>
-            <LinkBase
```

Comment:
We should be able to use the `DataList.Item` component here to handle the link / no-link cases automatically. I believe we didn't do that initially because previously we couldn't use the component outside the `DataList` context